docs/content/docs/connectors/datastream/elasticsearch.md

```diff
@@ -43,15 +43,11 @@ of the Elasticsearch installation:
   </thead>
   <tbody>
     <tr>
-        <td>5.x</td>
-        <td>{{< artifact flink-connector-elasticsearch5 >}}</td>
-    </tr>
-    <tr>
-        <td>6.x</td>
+        <td><= 6.3.1</td>
         <td>{{< artifact flink-connector-elasticsearch6 >}}</td>
     </tr>
     <tr>
-        <td>7 and later versions</td>
+        <td><= 7.5.1</td>
```

Comment:
Don't we support the entire 7.* series?

Reply:
Please see the comment above, I'll have to check if there is a dependency matrix or if we can confirm that with the docker image used in the tests